browse/templates/base.html

```diff
@@ -68,15 +68,15 @@
         {% block header %}
         {#- The ignore_me link is not meant to be visible to users; it is meant to catch robots/crawlers not respecting robots.txt. aria-hidden prevents screenreaders from being caught. -#}
         <a aria-hidden="true" href="{url_path('ignore_me')}"></a>
-        {% block header_h1 %}<h1>{{ config['BROWSE_SITE_LABEL'] or 'arXiv.org' }} <img src="{{ url_for('static', filename='images/arxiv-30.svg') }}" alt="30th anniversary" style="width:43px;float:right;margin:6px 0 0 12px;"/></h1>{% endblock header_h1%}
+        {% block header_h1 %}<h1><img src="{{ url_for('static', filename='images/arxiv-logo-one-color-white.svg') }}" alt="arxiv logo" style="height:60px;"/></h1>{% endblock header_h1%}
```

Comment:
Minor: is there any way we could easily distinguish between the main site and development sites?